frontend/test/__support__/mocks.js

```diff
@@ -4,8 +4,8 @@ global.ace.define = () => {};
 global.ace.require = () => {};
 
 global.window.matchMedia = () => ({
-  addListener: () => {},
-  removeListener: () => {},
+  addEventListener: () => {},
```

Comment:
addListener is not recommended anymore - [MDN](https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryList/addListener)